### ISSUE TITLE
track pa11y sidekick

### ISSUE THREAD

**Issue by andrewmallis** [kalamuna/pa11y-dashboard#11]
https://github.com/pa11y/sidekick

Pa11y Sidekick will be a modern and well thought out replacement for [Pa11y Dashboard] and [Pa11y Webservice]. What exactly is Pa11y Sidekick going to do? Still to be defined, but our rough proposal at this stage:

  > like Travis, but for a11y testing, and with graphs.

see https://github.com/pa11y/sidekick/blob/master/README.md for full details

**Comment by RobLoach** [comment]
Definitely, brought this up to Crispin a little bit ago, and have been keeping watch. Looks promising.
